bloom-housing/bloom
# 1653/user invite flow frontend

# Pull Request Template

## Issue

Addresses #1653


## Description

Adds the "Add user" drawer with form.

## Type of change


## How Can This Be Tested/Reviewed?

Use the newly created form to add a new user, then check if a user exists in the user table.
You can also test behavior if an email already exists (it should show a proper alert with an error message).


## Checklist:
